src/components/shared/Backtrace.js

```diff
@@ -46,7 +46,9 @@ function Backtrace(props: Props) {
         ))
       ) : (
         <li className="backtraceStackFrame">
-          (stack empty or all stack frames filtered out)
+          (The stack is empty because all its frames are filtered out by the
+          implementation filter. You can switch the implementation filter in the
+          call tree to see more frames.)
```

Comment:
We see the text is cut off because of the style. So maybe we should change this implementation a bit more:
* depending on the result of `funcNamesAndOrigins.length > 0` we could output a `<ol>` (like it is now) or a simple `div` with the text (and the same class `backtrace`).

I think it would be easier to use a normal `if` instead of the ternary, something like this:
```js
if (funcNamesAndOrigins.length) {
  return (
    <ol className="backtrace" style={{ '--max-height': maxHeight }}>
      {funcNamesAndOrigins.map(...)}
    </ol>
  );
} else {
  return (
    <div className="backtrace">
      ...message...
    </div>
  );
}
```

What do you think?